lib/reporters/xunit.js

```diff
@@ -130,7 +130,7 @@ XUnit.prototype.test = function(test) {
 
   if (test.state === 'failed') {
     var err = test.err;
-    this.write(tag('testcase', attrs, false, tag('failure', {}, false, cdata(escape(err.message) + '\n' + err.stack))));
```

Comment:
It was double escaping from the first cut: https://github.com/mochajs/mocha/pull/179/files#diff-d80a115516f0cc2ec0b46a54c60077faR88